src/main/scala/uk/gov/homeoffice/cirium/services/api/FlightScheduledRoutes.scala

```diff
@@ -0,0 +1,38 @@
+package uk.gov.homeoffice.cirium.services.api
+
+import akka.http.scaladsl.model.HttpResponse
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import com.typesafe.scalalogging.Logger
+import uk.gov.homeoffice.cirium.services.entities.{ CiriumScheduledFlightRequest, CiriumScheduledResponse }
+
+import scala.concurrent.Future
+
+trait FlightScheduledRoutes extends CiriumBaseRoutes {
+
+  import uk.gov.homeoffice.cirium.JsonSupport._
+
+  val logger = Logger(getClass)
+
+  val scheduleApiEndpoint = "https://api.flightstats.com/flex/schedules/rest/v1/json/flight"
+
+  val flightScheduledRoute = pathPrefix("flightScheduled") {
+    pathEndOrSingleSlash {
+      post {
```

Comment:
any reason why this is a post rather than a get?

Reply:
Good point . As i guess we are just getting data from cirium , we can do with GET . In fact it was GET before, I just change to POST as it was json  input. Let keep GET we are just getting data rather than modifying it here.